### PR TITLE
Add a Makefile script to install conftest

### PR DESCRIPTION
**Pull request by simonbaird**
Borrowed in part from #29 .

### CODE REVIEW

**Comment by simonbaird** [comment]
WIP since I'd like to get the CI working with conftest.

File: Makefile [hunk 1]
```diff
@@ -184,13 +196,38 @@ OPA_DEST=$(OPA_BIN)/opa
...
+  CONFTEST_BIN=$(HOME)/bin
+endif
+CONFTEST_DEST=$(CONFTEST_BIN)/conftest
+
+install-conftest: ## Install `conftest` CLI from GitHub releases
+	curl -s -L -O $(CONFTEST_URL)
+	echo "$(CONFTEST_SHA) $(CONFTEST_FILE)" | sha256sum --check
+	tar xzf $(CONFTEST_FILE) conftest
+	@mkdir -p $(CONFTEST_BIN)
+	mv conftest $(CONFTEST_DEST)
```
[the remark this answers is not in the document]
Comment (simonbaird) [reply Makefile:226]: Seems like it comes out of the tar ball with +x on it already. But I could just do that anyhow for peace of mind.

[the remark this answers is not in the document]
Comment (lcarva) [reply Makefile:226]: Sounds like a good idea.

[the remark this answers is not in the document]
Comment (simonbaird) [reply Makefile]: Done.

**Comment by simonbaird** [comment]
I threw in some readme updates here, rather than making a new PR.

**Review by lcarva — APPROVED** [review]
LGTM!

**Pull request merged**